packages/webpack-config/webpack/utils/getPathsAsync.js

```diff
@@ -31,6 +31,11 @@ module.exports = async function getPaths({ locations, projectRoot }) {
   const inputProjectRoot = projectRoot || appDirectory;
 
   function absolute(...pathComponents) {
+    // Simple check if we are dealing with an URL
+    if (pathComponents && pathComponents.length === 1 && pathComponents[0].startsWith('http')) {
```

Comment:
We should wait for #861 cuz this may break somethings.

Reply:
@EvanBacon could we consider this PR now after #861 was merged?